packages/ui/libs/certificate/view/src/containers/list/CertificateActionContent/CertificateActionContent.effects.ts

```diff
@@ -36,9 +37,10 @@ export const useCertificateActionContentEffects = <Id>(
               items.find((item) => item.id === id).energy.replace(/,/g, '')
             ).toString(),
           };
-          setEnergyAmounts([...energyAmounts, preparedNewItem]);
+          newItems.push(preparedNewItem);
         }
       });
+      setEnergyAmounts([...energyAmounts, ...newItems]);
```

Comment:
why do you change this logic? did you encounter any bug related to selection of items and this is the fix?

Reply:
Yes. When selecting multiple at once, only the value for the last one is taken into account. This was the fix for it.